test/test-cases/lint-rules/__snapshots__/import-order.test.js.snap

```diff
@@ -27,9 +27,9 @@ import c from '../c';
 import a from './a';
 import b from './b';
 
-import cTreat from '../c.treat';
 import aTreat from './a.treat';
 import bTreat from './b.treat';
+import cTreat from '../c.treat';
```

Comment:
Looks like we need to change _something_ in `eslint-config-seek` 🤔

Reply:
Isn't this likely a result of a change in our eslint config? https://github.com/seek-oss/sku/pull/735/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR7341
https://github.com/seek-oss/eslint-config-seek/releases/tag/v10.1.3

Reply:
Import ordering is a sku feature: 
https://github.com/seek-oss/sku/blob/cf67d47f0e109dafb0541a05c50311f56bd5baa9/config/eslint/importOrderConfig.js#L17-L39

Reply:
Weird that this changed then, unless something changed in how the eslint rule itself behaves.

Reply:
I think this is fine.